rest/models/tickers.go

```diff
@@ -267,71 +267,62 @@ type GetTickerTypesResponse struct {
 
 // Ticker contains detailed information on a specified ticker symbol.
 type Ticker struct {
-	Active                      bool           `json:"active"`
-	Address                     CompanyAddress `json:"address,omitempty"`
-	Branding                    Branding       `json:"branding,omitempty"`
-	CIK                         string         `json:"cik,omitempty"`
-	CompositeFIGI               string         `json:"composite_figi,omitempty"`
-	CurrencyName                string         `json:"currency_name,omitempty"`
-	DelistedUTC                 Time           `json:"delisted_utc,omitempty"`
-	Description                 string         `json:"description,omitempty"`
-	HomepageURL                 string         `json:"homepage_url,omitempty"`
-	LastUpdatedUTC              Time           `json:"last_updated_utc,omitempty"`
-	ListDate                    Date           `json:"list_date,omitempty"`
-	Locale                      string         `json:"locale,omitempty"`
-	Market                      string         `json:"market,omitempty"`
-	MarketCap                   float64        `json:"market_cap,omitempty"`
-	Name                        string         `json:"name,omitempty"`
-	PhoneNumber                 string         `json:"phone_number,omitempty"`
-	PrimaryExchange             string         `json:"primary_exchange,omitempty"`
-	ShareClassFIGI              string         `json:"share_class_figi,omitempty"`
-	ShareClassSharesOutstanding int64          `json:"share_class_shares_outstanding,omitempty"`
-	SICCode                     string         `json:"sic_code,omitempty"`
-	SICDescription              string         `json:"sic_description,omitempty"`
-	Ticker                      string         `json:"ticker,omitempty"`
-	TickerRoot                  string         `json:"ticker_root,omitempty"`
-	TickerSuffix                string         `json:"ticker_suffix,omitempty"`
-	TotalEmployees              int32          `json:"total_employees,omitempty"`
-	Type                        string         `json:"type,omitempty"`
-	WeightedSharesOutstanding   int64          `json:"weighted_shares_outstanding,omitempty"`
-}
-
-// CompanyAddress contains information on the physical address of a company.
-type CompanyAddress struct {
-	Address1   string `json:"address1,omitempty"`
-	Address2   string `json:"address2,omitempty"` // todo: add this to the spec
-	City       string `json:"city,omitempty"`
-	PostalCode string `json:"postal_code,omitempty"`
-	State      string `json:"state,omitempty"`
-}
-
-// Branding contains information related to a company's brand.
-type Branding struct {
-	LogoURL string `json:"logo_url,omitempty"`
-	IconURL string `json:"icon_url,omitempty"`
+	Active  bool `json:"active"`
+	Address struct {
```

Comment:
just seeing how people feel about this. i think i like it but i'm going to think about it over the weekend before merging.

Reply:
Were we defining the children structs outside of the parent struct before and now they've been moved inside? (Or were we not defining them at all before?)